src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.System/Helpers/Icons.cs

```diff
@@ -8,11 +8,11 @@ namespace Microsoft.CmdPal.Ext.System.Helpers;
 
 public static partial class Icons
 {
-    public static IconInfo FirmwareSettingsIcon { get; } = IconHelpers.FromRelativePaths("Microsoft.CmdPal.Ext.System\\Assets\\logoff.light.png", "Microsoft.CmdPal.Ext.System\\Assets\\logoff.dark.png");
+    public static IconInfo FirmwareSettingsIcon { get; } = new IconInfo("\uEBC4");
 
     public static IconInfo LockIcon { get; } = new IconInfo("\uE72E");
 
-    public static IconInfo LogoffIcon { get; } = IconHelpers.FromRelativePaths("Microsoft.CmdPal.Ext.System\\Assets\\logoff.light.png", "Microsoft.CmdPal.Ext.System\\Assets\\logoff.dark.png");
+    public static IconInfo LogoffIcon { get; } = new IconInfo("\uF3B1");
```

Comment:
before
![image](https://github.com/user-attachments/assets/9c7d703f-0745-495d-9cf5-09eb058f7c31)


after
![image](https://github.com/user-attachments/assets/e7c69921-b91e-4e9e-9191-ff221cce4041)

(the same)